ecleankernel/file.py

```diff
@@ -123,7 +123,14 @@ def decompress_raw(self) -> bytes:
                     # Technically a redundant import, this is just
                     # to make your IDE happy :)
                     import zstandard
-                    return zstandard.ZstdDecompressor().decompress(f.read())
+                    reader = zstandard.ZstdDecompressor().stream_reader(f)
+                    decomp = b''
+                    while True:
+                        chunk = reader.read(1024*1024)
+                        decomp += chunk
```

Comment:
Hmm, wouldn't it be better to move this after the `if` below? After all, appending an empty chunk doesn't seem to make much sense.

Reply:
good point - the `return decomp` was also underidented, whoops